React/Modules/RCTIPAddress.h

```diff
@@ -0,0 +1,3 @@
+// This file is generated by a build script.
+
+#define LOCAL_IP_ADDRESS @""
```

Comment:
We should not modify the sources. Consider taking the approach from #6362 - store the IP address into a file and load that file at runtime.